Add doc for neural-sparse-query-two-phase-processor. (#7306)
opensearch-project/documentation-website: _search-plugins/neural-sparse-search.md

  }
}
```
## Step 5: Create and enable two-phase processor (Optional)
'neural_sparse_two_phase_processor' is a new feature which introduced in OpenSearch 2.15. It can speed up the neural sparse query's time cost with negligible accurency loss
For more information, you can refer to [neural-sparse-query-two-phase-processor]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/neural-sparse-query-two-phase-processor/).

> 

Reviewer: Can we give some example API calling like the section `## Setting a default model on an index or field`? Using all default values is also good, this would help users a lot

Reviewer: We should also mention that this processor is strongly recommended to set

Author: Gave a example API for set a default 2-phase pipeline.  
Add a explanation for why we recommend this pipeline.

  }
}
```
## Step 5: Create and enable the two-phase processor (Optional)

The 'neural_sparse_two_phase_processor' is a new feature which introduced in OpenSearch 2.15. Using the two-phase processor can significantly improve the performance of neural sparse queries.

> To quickly launch a search pipeline with neural sparse search, use the following example: 

Reviewer 2: ```suggestion
To quickly launch a search pipeline with neural sparse search, use the following example pipeline: 
```

> To quickly launch a search pipeline with neural sparse search, use the following example: 

Reviewer 2: Added "pipeline".

```json
PUT /_search/pipeline/two_phase_search_pipeline
{
  "request_processors": [
    {
      "neural_sparse_two_phase_processor": {
        "tag": "neural-sparse",
        "description": "This processor is making two-phase processor."
      }
    }
  ]
}
```
{% include copy-curl.html %}

> Then choose the index you want to set up with the search pipeline and set the `index.search.default_pipeline` to the pipeline name, as shown in the following example:

Reviewer 2: ```suggestion
Then choose the index you want to set up with the search pipeline and set the `index.search.default_pipeline` to the pipeline name, as shown in the following example:
```
```suggestion
Then choose the index you want to configure with the search pipeline and set the `index.search.default_pipeline` to the pipeline name, as shown in the following example:
```

  }
}
```

> ## Step 5: Create and enable two-phase processor (Optional)

Reviewer 2: ```suggestion
## Step 5: Create and enable the two-phase processor (Optional)
```

> This step is optional but strongly recommended, as it significantly improves the performance of neural sparse queries with almost no side effects.

Reviewer 2: Going to remove this sentence for now.

> This step is optional but strongly recommended, as it significantly improves the performance of neural sparse queries with almost no side effects.

Reviewer 2: ```suggestion

```

> The 'neural_sparse_two_phase_processor' is a new feature which introduced in OpenSearch 2.15. It can speed up the neural sparse query's time cost with negligible accurency loss.

Reviewer 2: ```suggestion
The 'neural_sparse_two_phase_processor' is a new feature which introduced in OpenSearch 2.15. Using the two-phase processor can significantly improve the performance of neural sparse queries.
```
